src/parser/jobs/brd/modules/Repertoire.tsx

```diff
@@ -0,0 +1,86 @@
+import {Trans} from '@lingui/react'
+import Color from 'color'
+import {Event, Events} from 'event'
+import {filter} from 'parser/core/filter'
+import {CounterGauge, Gauge} from 'parser/core/modules/Gauge'
+import {GAUGE_FADE} from 'parser/core/modules/ResourceGraphs/ResourceGraphs'
+import React from 'react'
+import ACTIONS, {Action} from '../../../../data/ACTIONS'
+
+const MAX_WM_REPERTOIRE = 3
+const MAX_AP_REPERTOIRE = 4
+
+const SONG_GAUGE_INFO: {[key: number]: Action | undefined} = {
+	0x5: ACTIONS.MAGES_BALLAD,
+	0xA: ACTIONS.ARMYS_PAEON,
+	0xF: ACTIONS.THE_WANDERERS_MINUET,
+	0xC: undefined,
+}
+
+const WM_REPERTOIRE_COLOR = Color('#8BB8FF').fade(GAUGE_FADE)
+const AP_REPERTOIRE_COLOR = Color('#EDD78B').fade(GAUGE_FADE)
+
+export class Repertoire extends Gauge {
+	static override handle = 'repertoire'
+	private wmRepertoireGauge: CounterGauge | undefined
+	private apRepertoireGauge: CounterGauge | undefined
+
+	override initialise() {
+		super.initialise()
+		this.addEventHook(filter<Event>().actor(this.parser.actor.id).type('gaugeUpdate'), this.handleLoggedGauge)
+	}
+	private handleLoggedGauge(event: Events['gaugeUpdate']) {
+		// If we haven't yet noted that this player has gauge update events, set that now
+		if (!this.parser.actor.loggedGauge) { this.parser.actor.loggedGauge = true }
+
+		if ('repertoire' in event && event.repertoire != null) {
+			this.resourceGraphs.addDataGroup({
+				handle: 'repertoire',
+				label: <Trans id="brd.gauge.resource.repertoire">Repertoire</Trans>,
+				collapse: true,
+			})
+
+			switch (SONG_GAUGE_INFO[event.song]) {
+			case ACTIONS.THE_WANDERERS_MINUET:
+				if (!this.wmRepertoireGauge) {
+					this.wmRepertoireGauge = this.add(new CounterGauge({
+						maximum: MAX_WM_REPERTOIRE,
+						graph: {
+							handle: 'repertoire',
+							label: <Trans id="brd.gauge.resource.repertoire.wm">Wanderer's Repertoire</Trans>,
+							color: WM_REPERTOIRE_COLOR,
+							collapse: true,
```

Comment:
collapse on the individual gauge graphs isn't necessary since that's already set on the group